test-fixture/integration/src/main/kotlin/com/linecorp/kotlinjdsl/test/integration/querydsl/select/AbstractCriteriaQueryDslSelectIntegrationTest.kt

```diff
@@ -92,7 +92,7 @@ abstract class AbstractCriteriaQueryDslSelectIntegrationTest : AbstractCriteriaQ
     }
 
     @Test
-    fun `listQuery - select single subquery`() {
+    open fun `listQuery - subquery in select, subquery in from`() {
```

Comment:
Is there any reason to use open keyword only in this test?

Reply:
-- korean
원래라면 필요 없지만 eclipselink 가 subquery in select 를 지원하지 않기 때문에 이 메소드를 override 해서 예외에 대한 assertion을 구현할 필요가 있었습니다.
현재 테스트 구조는 Abstract 클래스들을 각 jpa 구현체가 그대로 상속했을때 테스트가 가능해야 하지만 eclipselink 의 경우 불가능했습니다.

-- english
Originally it was not necessary, but since eclipselink does not support subquery in select , I needed to override this method to implement assertion for exceptions.
The current test structure should be able to test when Abstract classes are inherited by each jpa implementation as it is, but in the case of eclipselink, this was not possible.

Reply:
-- korean
이 내용을 주석으로 남겨놓는 것이 좋겠습니다.

-- english
You'd better leave this as an comment.